# org_tensorflow 用の Android 向けパッチ

--- a/third_party/cpuinfo/cpuinfo.BUILD
+++ b/third_party/cpuinfo/cpuinfo.BUILD
@@ -257,7 +257,7 @@
 config_setting(
     name = "android_arm64",
     values = {
-        "crosstool_top": "//external:android/crosstool",
+        "crosstool_top": "@android_toolchain//:toolchain",
         "cpu": "arm64-v8a",
     },
     visibility = ["//visibility:public"],

--- a/tensorflow/BUILD
+++ b/tensorflow/BUILD
@@ -111,7 +111,7 @@ config_setting(
         {},
     ),
     values = if_oss(
-        {"crosstool_top": "//external:android/crosstool"},
+        {"crosstool_top": "@android_toolchain//:toolchain"},
         {},
     ),
     visibility = ["//visibility:public"],
@@ -230,7 +230,7 @@ config_setting(
     ),
     values = dict(
         if_oss(
-            {"crosstool_top": "//external:android/crosstool"},
+            {"crosstool_top": "@android_toolchain//:toolchain"},
         ),
         cpu = "arm64-v8a",
     ),
