diff --git a/tensorflow/lite/build_def.bzl b/tensorflow/lite/build_def.bzl
index 43015747773..d361e6f6c47 100644
--- a/tensorflow/lite/build_def.bzl
+++ b/tensorflow/lite/build_def.bzl
@@ -42,7 +42,7 @@ def tflite_copts():
     }) + select({
         clean_dep("//tensorflow:windows"): [],
         "//conditions:default": [
-            "-fno-exceptions",  # Exceptions are unused in TFLite.
+            # "-fno-exceptions",  # Exceptions are unused in TFLite.
         ],
     })
 
